quiz/templates/registeration.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %} {% block content %}
+
+<div class="col-md-6 offset-md-3">
+    <h1 class="text-center">Registeration Form</h1>
+    <form action= "{% url 'user' %}" method="POST">
+        {% csrf_token %}
+       {{ form.as_p }}
```

Comment:
instead of taking the entire form, we can have take the fields, so that designing can be done on this.